src/main/resources/db/migration/V017__add_new_column_reconciliation_report.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE envelope_reconciliation_reports
+ADD COLUMN report_for DATE DEFAULT NOW();
```

Comment:
do we need it or can use use date from related statement?

Reply:
mail service does not know supplier statement. sends the latest report and assume create_at is same as report_for date

Reply:
it doesn't need to.
fix the SQL query in `findByDate` repository method to use the statement column.